Doppler.Sap/Factory/ISapTaskHandler.cs

```diff
@@ -6,7 +6,7 @@ namespace Doppler.Sap.Factory
     public interface ISapTaskHandler
```

Comment:
I think that interface is not necessary, can you remove and check it?

Reply:
We need the interface for the unit tests